src/bp/ui-shared/src/ContentForms/utils/fields.ts

```diff
@@ -243,13 +20,14 @@ export const getEmptyFormData = (contentType: string, isPartOfGroup = false): Fo
         text: '',
         buttons: []
       }
-    case 'carousel':
+    case 'builtin_carousel':
       return {
         markdown: true,
         typingIndicator: true,
-        cards: [getEmptyFormData('card', true)]
+        cards: [getEmptyFormData('builtin_card', true)]
       }
     case 'suggestions':
```

Comment:
Shouldn't this one be removed?

Reply:
No it's still referenced in the form as it's the "field" name and it's what's used to generate the empty data